CraftTweaker2-API/src/main/java/crafttweaker/api/entity/IEntity.java

```diff
@@ -505,12 +509,13 @@ default IRayTraceResult getRayTrace(double blockReachDistance, float partialTick
     }
     
     @ZenMethod
-    default void update(IData data){
+    default void update(IData data) {
         CraftTweakerAPI.logError("IEntity#update not overwritten by implementation " + this.getClass() + "!");
     }
-    
+
+    @ZenSetter("nbt")
     @ZenMethod
-    default boolean onGround(){
-        return false;
```

Comment:
Usually I would say that since this method is already in the interface, it shouldn't be removed, but since it was added only recently by you, it should be fine.